templates/plugins/generic/pdfJsViewer/templates/display.tpl

```diff
@@ -29,39 +29,38 @@
 	{load_script context="frontend" scripts=$scripts}
 </head>
 <body class="pkp_page_{$requestedPage|escape} pkp_op_{$requestedOp|escape}">
+	{include file="frontend/components/header.tpl"}
 
-	{* Header wrapper *}
-	<header class="main__header pdf-galley__header">
+	<main class="container galley">
+		<div class="row">
+			<header class="offset-lg-1 col-lg-10 galley__header">
+				<p>
+					<a href="{$parentUrl}" class="btn btn-secondary">
+						&larr;
+						{if $parent instanceOf Issue}
+						{translate key="issue.return"}
+						{else}
+						{translate key="article.return"}
+						{/if}
+					</a>
 
-		<div>
-			<a href="{$parentUrl}">
-				←
-				<span class="sr-only">
-					{if $parent instanceOf Issue}
-					{translate key="issue.return"}
-					{else}
-					{translate key="article.return"}
-					{/if}
-				</span>
-				{$title}
-			</a>
-		</div>
-		
-		<div>
-			<a href="{$pdfUrl}" class="btn btn-primary" download>
-				{translate key="common.download"}
-				<span class="sr-only">
-					{translate key="common.downloadPdf"}
-				</span>
-			</a>
-		</div>
+					<a href="{$pdfUrl}" class="btn btn-primary float-right" download>
+						{translate key="common.download"}
+						<span class="sr-only">
+							{translate key="common.downloadPdf"}
+						</span>
+					</a>
+				</p>
+				<h1>{$article->getLocalizedTitle()}</h1>
+			</header>
 
-	</header>
-
-	<div id="pdfCanvasContainer">
-		<iframe src="{$pluginUrl}/pdf.js/web/viewer.html?file={$pdfUrl|escape:"url"}" width="100%" height="100%" style="min-height: 500px;" allowfullscreen webkitallowfullscreen></iframe>
-	</div>
+			<div id="pdfCanvasContainer" class="offset-lg-1 col-lg-10 galley__content">
```

Comment:
There can be a problem in scrolling for iOS devices. Take a look at this related issue: https://github.com/pkp/healthSciences/issues/147

Reply:
Done!